### ISSUE TITLE
x64 build is arm64, not x86_64 :(

### ISSUE THREAD

**Issue by subtleGradient** [jamsocket/y-sweet#270]
```
$ uname -m
x86_64

$ file y-sweet-macos-x64
y-sweet-macos-x64: Mach-O 64-bit executable arm64
```

may I have an x86_64 build please?

**Comment by subtleGradient** [comment]
successfully installed y-sweet on my late 2014 mac mini (pet server) using cargo (very slowly) but it works!
yay!

**Comment by paulgb** [comment]
Thanks for reporting, @subtleGradient! I've merged #271, which will fix this as of our next numbered release (should happen this week).
